tensorflow/core/kernels/conv_grad_filter_ops.cc

```diff
@@ -980,6 +992,106 @@ void LaunchConv2DBackpropFilterOp<Eigen::GpuDevice, T>::operator()(
   // if we do not have a cached algorithm_config for this conv_parameters
   cudnn_use_autotune = true;
 #endif
+#if GOOGLE_CUDA && CUDNN_VERSION >= 8100
```

Comment:
Instead of always use the frontend plan, can you add switch (e.g. a env var `TF_DEBUG_CUDNN_FRONTEND`) to turn it on, and off by default? This would help our internal benchmarking.

Reply:
Done.